Latex allows to force placements via !.

schrc3b6 opened the pull request:
Added new terminal for ! and the nonterminal P for placements.
Added P to figures and tables.

schrc3b6 committed:
Latex allows to force placements via !.
Added new terminal for ! and the nonterminal P for placements.
Added P to figures and tables.

pkubowicz commented:
Thank you for the PR.

Could you add a test demonstrating the new functionality?

Usually this means appending some LaTeX code to `test/in.tex` and expected output to `test/correct.txt`. For example, see 646143802eb7b.

schrc3b6 committed:
Changed placement to allow the exclamation point anywhere as documented in:
"How to influence the position of float environments like figure and table in LATEX?"

"The order in which these characters are put in
the optional argument does not influence how the
algorithm tries to place the float!"

schrc3b6 committed:
added ! in both table and figures

schrc3b6 commented:
I added ! to both table and figure placement in in.tex. I reused the figure and table already in the tex file.
